test/integration/serverless_perf/write_message.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+filename="${HOME}/comment.md"
+echo "filename=$filename" >> "$GITHUB_OUTPUT"
+
+if [ "${VAR_COLD_START}" != 0 ]; then
```

Comment:
What are the values that `VAR_COLD_START` usually store?

Reply:
The value is defined [here](https://github.com/DataDog/datadog-agent/blob/rey.abolofia/bin-size-info/.github/workflows/serverless-binary-size.yml#L77-L79)

```
          OUTPUT=$(( $OUTPUT / 100000 ))
          echo "cold start time changed by $OUTPUT ms"
          echo "coldstart=$OUTPUT" >> $GITHUB_OUTPUT
```

Therefore, the value is always a positive integer.

For example:

```bash
$ echo $(( 1 / 10 ))
0
$ echo $(( 9 / 10 ))
0
$ echo $(( 10 / 10 ))
1
$ echo $(( 20 / 10 ))
2
```